src/sass/main.scss

```diff
@@ -1,24 +1,16 @@
+
 // OVERWRITES
```

Comment:
I think it would be cool to show of some overwrites in this starter kit to show new users how the colour system, spacing and typography works. Even if they are commented out.